codecademy/introduction_to_classes.py

```diff
@@ -4,19 +4,22 @@ def __init__(self, name, color, flavor, poisonous, weight):
         self.name = name
         self.color = color
         self.flavor = flavor
+        assert type(poisonous) == bool, "Error type, please enter boolean for poisonous"
         self.poisonous = poisonous
         self.weight = weight
 
     def description(self):
         return "I'm a %s %s and I taste %s." % (self.color, self.name, self.flavor)
 
-    def is_edible(self, type_posinous):
-        assert type(type_posinous) == bool, "Error type, please enter boolean"
+    def is_edible(self):
```

Comment:
@09041985 i missed this problem when it was introduced.
Fixed it here.
is_edible() is pretty dumb method, it only checks for the self.poisonous and returns the opposite boolean value.
It should not need any arguments. (see for yourself that type_posinous argument is not being used meaningfully.